middle_end/flambda2/to_cmm/to_cmm_shared.ml

```diff
@@ -426,3 +426,15 @@ let extended_machtype_of_return_arity arity =
   | arity ->
     (* Functions returning multiple values *)
     List.map extended_machtype_of_kind arity |> Array.concat
+
+let alloc_mode_to_cmx t =
```

Comment:
Maybe `alloc_mode_for_applications_to_cmx` would be clearer, and similarly for the other function below.